maven-embedder/src/test/java/org/apache/maven/cli/event/ExecutionEventLoggerTest.java

```diff
@@ -119,16 +121,12 @@ public void testProjectStartedOverflow()
         inOrder.verify( logger ).info( "" );
         inOrder.verify( logger ).info( "--< org.apache.maven.plugins.overflow:maven-project-info-reports-plugin >--" );
         inOrder.verify( logger ).info( "Building Apache Maven Project Info Reports Plugin 3.0.0-SNAPSHOT" );
-        inOrder.verify( logger ).info( "  from pom.xml" );
+        inOrder.verify( logger ).info( adaptDirSeparator( "  from pom.xml" ) );
```

Comment:
From a cleanliness perspective I would apply this only to "pom.xml" and not to "from ".